src/libbpfilter/cli.c

```diff
@@ -7,13 +7,195 @@
 #include <string.h>
 
 #include "core/chain.h"
+#include "core/counter.h"
```

Comment:
I'll skip this one for now to focus on the daemon side, but some thoughts: functions to dump the chains should be in `bfcli/**.c`, not in `libbpfilter` which should focus on communication with the daemon.